### PR TITLE
update tracker list

### PR DESCRIPTION
**Pull request by ddogfoodd**

### CODE REVIEW

**Comment by Tienisto** [comment]
It seems that name and website has been swapped. Have you tested this?

**Comment by ddogfoodd** [comment]
Noticed but not tested, I assumed json is used as a dictionary and its elements are accessed via the corresponding key. If they are accessed via index, this will lead to problems.
I can check/test this later today.

**Comment by Tienisto** [comment]
Okay, maybe my Ghidra implementation will do some trouble

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The analyse duration will be much longer but maybe ghidra finds more hidden trac
 
 If you get a **Permission Denied** error, then you should make them executable:
 
-`chmod -R +x <script location>/dependencies`
+`chmod -R +x <script location>/dependencies>`
```
Comment (Tienisto) [inline README.md]: I think the `>` was already closed

Comment (ddogfoodd) [reply README.md]: my bad

Comment (ddogfoodd) [reply README.md]: I reversed the last commit

**Review by Tienisto — APPROVED** [review]

**Pull request merged**